common/src/models/deployment.rs

```diff
@@ -47,6 +56,48 @@ impl Display for Response {
     }
 }
 
+impl Display for EcsResponse {
```

Comment:
suggestion: IMHO we shouldn't use `Display` for that, as it includes color, which is something that only concern the CLI display. Here it means that if we log this value, we will get an output with escape code.

Reply:
Good thought. I'll add this logic outside of `Display` then, which will be the same except for the coloring.